packages/office-ui-fabric-react/src/components/Calendar/CalendarPage.tsx

```diff
@@ -26,10 +25,10 @@ export class CalendarPage extends React.Component<IComponentDemoPageProps, any>
             </ExampleCard>
             <ExampleCard title='Inline Calendar with overlayed month picker when header is clicked' code={ CalendarInlineExampleCode }>
               <CalendarInlineExample showMonthPickerAsOverlay={ true } highlightCurrentMonth={ true } dateRangeType={ DateRangeType.Day } autoNavigateOnSelection={ false }
-                showGoToToday={ true } />
+                showGoToToday={ false } />
             </ExampleCard>
             <ExampleCard title='Inline Calendar with month picker' code={ CalendarInlineExampleCode }>
-              <CalendarInlineExample dateRangeType={ DateRangeType.Day } autoNavigateOnSelection={ false }
+              <CalendarInlineExample dateRangeType={ DateRangeType.Day } autoNavigateOnSelection={ false } highlightCurrentMonth={ true }
```

Comment:
I suggest you add a test for when the first day of week is something other than the default and you are using a custom date range like month, to make sure the corners are correct

Reply:
Hi!  Not sure what you mean, the function that looks for the month corners only runs when the DateRanteType === 2 (month), it wouldn't run with DateRangeType.Day.  Lets discuss Monday!

Reply:
This is tested and working with the updated from 8-30-17